README.rst

```diff
@@ -748,8 +748,8 @@ All other default values are highlighted with bold:
    * - Philippines
      - PH
      -
-     -
-     -
+     - **en_PH**, en_US, th, tl
```

Comment:
We need to figure out `fil` vs `tl` question. I added a comment [here](https://github.com/vacanza/holidays/issues/2067#issuecomment-2451107369). @KJhellico WDYT?

Reply:
I'm fine either way - alternatively, we can add both `fil` and `tl` for now as well 👀

Reply:
As `fil` is officially registered by ISO, and its difference from Tagalog is officially declared, I support `fil`.

Reply:
I'll go with `fil` only and remove `tl` then in that case 👍

Reply:
> I'll go with `fil` only and remove `tl` then in that case 👍

Sounds good to me. Should it be `fil_PH`?

Reply:
Maybe just `fil`? I don't think there are any other `fil` variation...

Reply:
I still haven't figured the locales definition/format out properly, tbh. But:

```
cat /etc/locale.gen | grep -i fil
# This file lists locales that you wish to have built. You can find a list
# this file, you need to rerun locale-gen.
# fil_PH UTF-8
```

Reply:
Hmm - I did run them on Poedit earlier just to make sure and `fil` is a legit code, I don't think we need to add `_PH` suffix here
![image](https://github.com/user-attachments/assets/ea5d9cc3-9950-41c5-b432-1db3124b9bb9)

Reply:
I don't have an opinion on this. Let's assume `fil` is okay for now and improve it later if needed.

Reply:
We can perhaps keep this for future v1 standardization whether if language codes alone are ok or do we want to enforce locale code as well. :eyes: